content/release-notes/1.20.0.md

```diff
@@ -22,24 +22,24 @@ kubernetes: "1.17, 1.18, and 1.19"
   - Check for updates button is always available.
   - Current deployment status is included.
 * Added resource limits and requests for Admin Console pods.
-* Admin Console's will use `NODE_PORT` environment variable if set (e.g. from kURL embedded clusters), instead of default 8800.
-* Port forwarding no longer uses the `app.k8s.io/v1beta1` Application spec and has been aligned with [documentation](/vendor/config/dashboard-buttons/#provide-a-kots-application-spec).
+* Admin Consoles will use `NODE_PORT` environment variable if set (e.g. from kURL embedded clusters), instead of the default 8800.
+* Port forwarding no longer uses the `app.k8s.io/v1beta1` Application spec and has been aligned with the [documentation](/vendor/config/dashboard-buttons/#provide-a-kots-application-spec).
 * [SupportBundle](https://troubleshoot.sh/docs/support-bundle/collecting/) and [Preflight](https://troubleshoot.sh/docs/preflight/introduction/) specs are no longer served on unauthenticated endpoints and instead are provided to [support-bundle and preflight binaries](https://troubleshoot.sh/docs/#installation) via [secrets](/kotsadm/troubleshooting/support-bundle/).
-* Version check now correctly reports current application version and adds information about KOTS Admin Console version, Kubernetes version, app status, etc.
+* The version check endpoint now correctly reports the current application version and adds information about KOTS Admin Console version, Kubernetes version, app status, etc.
 {{</changes>}}
 
 {{<fixes>}}
-* Airgap bundles can now only be installed if the app matches the license.
+* When an airgap bundle that does not match the installed license is uploaded, show an error message that explicitly says this instead of "failed to verify bundle signature".
```

Comment:
It's always been the case that you'll fail to install bundles from different apps than the license
It's just the error message was horrible: https://github.com/replicatedhq/kots/pull/1096